### PR TITLE
Nick/secdir2

### PR DESCRIPTION
**Pull request by grittygrease**

### CODE REVIEW

**Comment by seanturner** [comment]
A reference QUIC would also need to be added to the references section  If it's an informative draft maybe not a thing, but if it's a normative reference then this draft will sit in the queue until QUIC transport gets published.  Maybe that's okay, but just an FYI.

**Review by seanturner — APPROVED** [review]

File: draft-ietf-tls-exported-authenticator.md [hunk 1]
```diff
@@ -124,17 +124,22 @@ SHOULD use TLS as its underlying transport to keep the request confidential.  Th
...
+The CertificateRequest and ClientCertificateRequest messages are used to define the
+parameters in a request for an authenticator.  These messages do not include any
+TLS framing and are not encrypted with a handshake key.
+
+The structures are defined to be:
+
+       struct {
+          opaque certificate_request_context<0..2^8-1>;
+          Extension extensions<2..2^16-1>;
+       } ClientCertificateRequest;
```
Comment (kaduk) [inline draft-ietf-tls-exported-authenticator.md:142]: If we don't use the handshake-layer framing or allocate a handshake message type for this, is there really any value in giving a new named structure?

Comment (martinthomson) [reply draft-ietf-tls-exported-authenticator.md:142]: My impression of the discussion in Singapore was that we would define a new handshake message type for this.  Maybe I'm misremembering though.  It was a long week.

Comment (chris-wood) [reply draft-ietf-tls-exported-authenticator.md:142]: I think the outcome was that there'd be no new handshake message type, so I think this is fine as is.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**